### ISSUE TITLE
The picker does not work during server side rendering.

### ISSUE THREAD

**Issue by iarvind** [coderhaoxin/react-flatpickr#9]
I see this error during startup of my server.
ReferenceError: document is not defined
    at Object.<anonymous> (/node_modules/flatpickr/dist/flatpickr.js:1571:22)

 **Is there a way where I don't have to do check for `(typeof window !== 'undefined')`**

**Comment by coderhaoxin** [comment]
Can you give me a code snippet to reproduce this? (I don't use server side rendering personally ((^_^))!!~~)

**Comment by snobojohan** [comment]
In a project that has server rendering I use it only in browser, since we only show it after users have clicked. What we do is check if we are in the browser when importing.

```
import {isBrowser} from '../../utils/environment';
const Flatpickr = isBrowser() ? require('react-flatpickr') : () => false;
```

So now when using `<Flatpickr />` it's nothing when server rendering. The `isBrowser()` function is very simple.

```
export function isBrowser() {
    return typeof window === 'object';
}
```

**Comment by nfriedly** [comment]
This is now fixed in flatpickr v3.0.7; I just sent #40 to update react-flatpickr to the latest release.

**Comment by redbmk** [comment]
There seems to have been some regression here. I think it's due, at least in part, to these two lines:

`window.flatpickr = flatpick` ([`index.ts`](https://github.com/chmln/flatpickr/blob/cfedaee72eb8d364f6fa24648165005a760dc312/src/index.ts#L2602))

`animate: window && window.navigator.userAgent.indexOf("MSIE") === -1,` ([`types/options.ts`](https://github.com/chmln/flatpickr/blob/cfedaee72eb8d364f6fa24648165005a760dc312/src/types/options.ts#L285))
